symfony/phpunit-bridge/4.1/.env.test

```diff
@@ -3,3 +3,4 @@ KERNEL_CLASS='App\Kernel'
 APP_SECRET='$ecretf0rt3st'
 SYMFONY_DEPRECATIONS_HELPER=999999
 PANTHER_APP_ENV=panther
+PANTHER_ERROR_SCREENSHOT_DIR=./tests/error-screenshots
```

Comment:
What about `./var/error-screenshots`? It's already ignored

Reply:
Isn't it harder to discover? I was hesitating with simply `./error-screenshots`, at the root.

Reply:
I like `var/something` also. Doc will tell where this is anyway.

Reply:
@nicolas-grekas yes, I updated the PR accordingly. It looks like my comment saying that I changed my mind has been lost :/ I have issues with GitHub these days.